cmd/status.go

```diff
@@ -0,0 +1,308 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"sort"
+	"strings"
+
+	"github.com/Masterminds/semver"
+	"github.com/fatih/color"
+	"github.com/mattn/go-isatty"
+	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/elastic/elastic-package/internal/cobraext"
+	"github.com/elastic/elastic-package/internal/packages"
+)
+
+var (
+	// taken from https://github.com/fatih/color/blob/4d2835ff85a014514ee435d49f76dc8b25c9cee3/color.go#L20-L21
+	noColor = os.Getenv("TERM") == "dumb" ||
+		(!isatty.IsTerminal(os.Stdout.Fd()) && !isatty.IsCygwinTerminal(os.Stdout.Fd()))
+)
+
+const (
+	productionURL = "https://epr.elastic.co"
+	stagingURL    = "https://epr-staging.elastic.co"
+	snapshotURL   = "https://epr-snapshot.elastic.co"
+)
+const statusLongDescription = `Use this command to display the current deployment status of a package.
+
+If a package name is specified, then information about that package is
+returned, otherwise this command checks if the current directory is a
+package directory and reports its status.`
+
+func setupStatusCommand() *cobraext.Command {
+	cmd := &cobra.Command{
+		Use:   "status [package]",
+		Short: "Show package status",
+		Args:  cobra.MaximumNArgs(1),
+		Long:  statusLongDescription,
+		RunE:  statusCommandAction,
+	}
+	cmd.Flags().BoolP(cobraext.NoColorFlagName, "c", false, cobraext.NoColorFlagDescription)
```

Comment:
I think this logic is not required. I don't see a use case that developer would like to disable coloring. If so, there is the flag `TERM`. It would be for removing this.